compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -410,6 +410,8 @@ prettyScenarioErrorError (Just err) =  do
           scenarioError_ContractIdInContractKeyKey
     ScenarioErrorErrorComparableValueError _ ->
       pure "Attend to compare incomparable values"
+    ScenarioErrorErrorTooDeepValueError _ ->
```

Comment:
Can we include the actual and the max nesting value in the errors

Reply:
I added the nesting statically as it would be in the spec. 
For the actual nesting of the value : 
- I am not sure it is not very useful. 
- We have to compute it from the root of the value (and not from were we throw), This new method to compute the nesting of a value need to be stack safe, and we have to pass it through the proto. I think it is not worthwhile.